### DISCUSSION TITLE
Recent news on Overture Maps foundation

### DISCUSSION THREAD

**Discussion by revosw** [fleaflet/flutter_map#1418] [General]
Just wanted to share the newly announced [Overture Maps foundation](https://overturemaps.org/). From what I have read, the foundation is not competing with OpenStreetMap. See the "Why overture?" section on their website. Excited to see the direction of flutter_map given even more types of map data

https://www.linuxfoundation.org/press/linux-foundation-announces-overture-maps-foundation-to-build-interoperable-open-map-data

**Comment by JaffaKetchup** [comment]
Looks interesting, thanks for letting us know!
Personally, I have mixed feelings about this: I'm not too comfortable having so many big tech corporations as founders (look at me being a hypocrite, using Flutter :D), and I'm not sure how _actually_ easy this will be to use. Open-ness is great, and I myself regularly contribute to OpenStreetMap, but having to pay to contribute is a bit odd? Maybe I don't fully understand.

In terms of the flutter_map direction with this, it looks like this will be more raw data instead of tiles, so I'm not sure what will change. Maybe some more plugins, depending on the data they release?
